pkg/apis/triggers/v1alpha1/cluster_trigger_binding_validation_test.go

```diff
@@ -30,23 +30,49 @@ func Test_ClusterTriggerBindingValidate(t *testing.T) {
 		tb   *v1alpha1.ClusterTriggerBinding
 	}{{
 		name: "empty",
-		tb:   bldr.ClusterTriggerBinding("name"),
+		tb: &v1alpha1.ClusterTriggerBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "name",
+			},
+		},
 	}, {
 		name: "multiple params",
-		tb: bldr.ClusterTriggerBinding("name",
-			bldr.ClusterTriggerBindingSpec(
-				bldr.TriggerBindingParam("param1", "$(body.input1)"),
-				bldr.TriggerBindingParam("param2", "$(body.input2)"),
-				bldr.TriggerBindingParam("param3", "$(body.input3)"),
-			)),
+		tb: &v1alpha1.ClusterTriggerBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "name",
+			},
+			Spec: v1alpha1.TriggerBindingSpec{
+				Params: []v1alpha1.Param{{
+					Name:  "param1",
+					Value: "$(body.input1)",
+				}, {
+					Name:  "param2",
+					Value: "$(body.input2)",
+				}, {
+					Name:  "param3",
+					Value: "$(body.input3)",
+				}},
+			},
+		},
```

Comment:
I see this PR mainly focuses on status related changes
So can we move these changes to [Migrate from test builders to structs](https://github.com/tektoncd/triggers/pull/1080) ?

Reply:
yeah this PR builds on the other PR. so once that is merged, I'll rebase. Otherwise, there were a bunch of conflicts.